Lo-Agency/polaris
# member entity

# Description

This PR fix member entity and send invitation link to new users.

## Related issues
Fixes [#144](https://quire.io/w/Caspian_Tamin/144/member_entity?view=board)

## Type of change


# How Has This Been Tested?

1. Add member and check the database
2. Get invite email
3. check the link as loged out user, login user and user without account.

# Checklist:
